### DISCUSSION TITLE
Saving crop parameters and backgorund color

### DISCUSSION THREAD

**Discussion by francescofugazzi** [NVlabs/instant-ngp#428] [Q&A]
Hi,

is it possible to save crop parameters and background colors? And is there anyway to pass those paramenters when loading back a scene through testbed or run.py?
Maybe this can be done with other parameters as well.

Thanks

**Comment by Tom94** [comment]
Hi there. You can add corresponding lines of Python code (e.g. `testbed.background_color = (R, G, B, A)`) to `run.py` at whichever place is most convenient to your application (e.g. before training starts). See `src/python_api.cu` for a list of all bindings.

**Comment by francescofugazzi** [comment]
thank you for your reply. I'm not a coder but i can manage that. 
About setting the min and max x,y,z crop is this the binding to refer to? 
Testbed::m_aabb

**Reply by Tom94** [reply]
`Testbed::m_render_aabb` is the one. :)

**Reply by francescofugazzi** [reply]
thank you :)

**Reply by francescofugazzi** [reply]
unfortunately the m_render_aabb is not a funcrion that support arguments through CLI :(
